target/linux/ramips/image/lzma-loader/src/Makefile

```diff
@@ -23,16 +23,16 @@ FLASH_START	:=
 FLASH_OFFS	:=
 FLASH_MAX	:=
 PLATFORM	:=
-CACHE_FLAGS	:=
+SUBTARGET	:=
+CACHE_FLAGS	:= -DCONFIG_ICACHE_SIZE="(32 * 1024)" \
+		   -DCONFIG_DCACHE_SIZE="(16 * 1024)" -DCONFIG_CACHELINE_SIZE=32
```

Comment:
While you are only working on 'step 1' of the unification (just for ramips, where I'm hoping step 2 will be that it will be generic enough for openwrt as a whole) I wonder if this is the best place fort his. Can we not get (and otherwise set) this in a makefile higher? idk

Reply:
Moved up a Makefile level